code/game/objects/items/stunbaton.dm

```diff
@@ -18,6 +18,7 @@
 	var/hitcost = 1000
 	var/throw_hit_chance = 35
 	var/preload_cell_type //if not empty the baton starts with this type of cell
+	var/resistance_chem = /datum/reagent/medicine/epinephrine

```

Comment:
We already have a ton of chems that handle stuns and stamina, it makes zero sense to use epinephrine for this one. Not to mention that every player starts with an epipen in their backpack.

Reply:
Really disagree, having it available is the entire point. If people aren’t want ping this specifically then I’ll close PR/change chem.